tests/TestUtility/TestHelpers.cs

```diff
@@ -5,6 +5,8 @@
 using Microsoft.CodeAnalysis.CSharp;
 using OmniSharp;
 using OmniSharp.Services;
+using System.IO;
+using System.Reflection;
```

Comment:
actually they are not needed at all, the snuck in there due to some code that ended up being removed